backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/steps/Store.kt

```diff
@@ -44,7 +44,7 @@ class Store(private val logger: Logger, private val manualImport: Boolean) : Pip
                     PhysicalStoreEntity.new {
                         storeId = storeEntity.id
                         addressId = address.id
-                        coordinates = Point(acceptingStore.longitude, acceptingStore.latitude)
+                        coordinates = Point(acceptingStore.longitude!!, acceptingStore.latitude!!)
```

Comment:
Why are these not null here?

Reply:
Because they are filtered out in the `Sanitize` step where your other comment was. If we forget to do that, an exception is thrown here and the current store is skipped/not inserted in the database